tools/saas_metrics_exporter/commit_distance/commit_distance.py

```diff
@@ -26,6 +26,9 @@ class DistanceKey:
     ref_from: str
     ref_to: str
 
+    def __hash__(self) -> int:
+        return hash((self.repo_url, self.ref_from, self.ref_to))
```

Comment:
how about declare `auth_code` field not used for hash or compare? like `auth_code: HasSecret | None = field(hash=False, compare=False)`

Reply:
Good idea. Fixed